### PR TITLE
More 0.6 fixes

### PR DESCRIPTION
**Pull request by andreasnoack**

### CODE REVIEW

File: src/RNGTest.jl [hunk 1]
```diff
@@ -1,7 +1,6 @@
 module RNGTest
 
     using Compat
-    import Compat.String
```
Comment (tkelman) [inline src/RNGTest.jl]: why did you delete this, it introduces a bunch of depwarns on 0.4

best to drop 0.4 or put this back

Comment (andreasnoack) [inline src/RNGTest.jl]: Thanks. I didn't notice that. In my recollection, this was a 0.3 requirement. I'll add it back and retag.

Comment (tkelman) [inline src/RNGTest.jl]: if only we could make travis statuses orange if there are depwarns...
